setup.py

```diff
@@ -25,6 +25,9 @@
     'Programming Language :: Python :: 3.6',
     'Framework :: Django :: 1.10',
     'Framework :: Django :: 1.11',
+    'Framework :: Django :: 2.1',
```

Comment:
but `Django :: 2.0`is still missing. Although deprecated, but still relevant.

Reply:
Now it's good. Sorry for delay and the futility of the problem.